src/utils/numeral.ts

```diff
@@ -6,7 +6,7 @@ import {Lang, i18n} from './i18n';
 // Set space delimiter for all locales possible in project
 Object.values(Lang).forEach((value) => {
     if (numeral.locales[value]) {
-        numeral.locales[value].delimiters.thousands = ' ';
+        numeral.locales[value].delimiters.thousands = '\xa0';
```

Comment:
looks like \xa0 should be some global constant 
<img width="200" alt="image" src="https://github.com/user-attachments/assets/691c9648-308e-42fd-9760-a859320359ce">